src/prefect/tasks/aws/s3.py

```diff
@@ -43,6 +43,7 @@ def run(
         credentials: str = None,
         bucket: str = None,
         compression: str = None,
+        raw: bool = False,
```

Comment:
I think I would prefer `as_bytes` but it may be worth waiting for @jcrist to weigh in.

Reply:
Yeah, `as_bytes` would be preferred here. This probably should have been the default originally, but that ship has sailed.